### DISCUSSION TITLE
Wordfence Blocking Imports

### DISCUSSION THREAD

**Discussion by sgibson71** [code4recovery/12-step-meeting-list#795] [General]
### Contact Details

sgibson71@gmail.com

### Website With Issue

https://pa-al-anon.org/

### What happened?

Running the meeting finder on a site along with Wordfence.  Everything seems fine for a period of time and then after about a week imports start to stall and take FOREVER to run.  Through trial and error we have found that deactivating Wordfence resets the issue and the imports will run normally for a period of time.  Not sure if this is a Meeting Finder issue or something Wordfence is doing wrong.  If you have any ideas I'd be interested in hearing them.

Thanks for developing this amazing tool - yours in service, Scott

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari, Microsoft Edge

**Comment by 97eighty** [comment]
@sgibson71  How often and how many imports do you do? Does the problem ever occur with WordFence deactivated?

**Comment by sgibson71** [comment]
This is what our chairperson had to say:

On average, I would say we do about 10 imports per week. Marge in group
records does 5-7 a week, and many of them require some manual overrides by
Lori or me, which we do with another load. So many days, there are two
loads done. Marge often starts her about 10 in the morning. Lori or I will
usually do the second load in the evening sometime. Each load right now is
416 meetings. And no, we've never experienced the problem while WordFence
is deactivated. If the load is having a problem, as soon as I deactivate
WordFence, the load immediately takes off and finishes in its usual 3-5
minutes. I can then reactivate WordFence and right now, we appear to be
good for a week. The slowdown also appears gradually, hitting a
"knee-in-the-curve" by Monday.

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Wed, Apr 27, 2022 at 1:17 PM 97eighty ***@***.***> wrote:

> @sgibson71 <https://github.com/sgibson71> How often and how many imports
> do you do? Does the problem ever occur with WordFence deactivated?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/issues/791#issuecomment-1111265740>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASEV4OYCILA2A6FRSOUM5WDVHFZCNANCNFSM5UPJAA4Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

*MOBILE:* (717) 333-4277
*WEB: *www.axiomwebworks.com

**Comment by 97eighty** [comment]
@sgibson71 Great information. Thanks. WordFence has a throttling and rate limiting feature built in. It's possible this feature is causing problems. Can you review the settings to see if there are any limitations? Unfortunately, WordFence doesn't allow you to bypass for a specific plugin. But taking a look at these settings and adjusting may help or at least identify the cause. Take a look at this WordFence video on YouTube to understand and access this feature. 

https://youtu.be/9mtGKVZcZAU

If this isn't the cause we'll have to do some more investigation.

**Comment by sgibson71** [comment]
Interesting video.  I had thought this might be the case but I'm not en
expert on wordfence.  I've attached a screenshot, to my understanding it's
not really set to throttle anything as it is, and I even reactivated the
learning mode to see if we could capture the traffic that's causing the
issue.

Is it possible that we need to set some exclude directories to stop
throttling on the import function?
https://www.wordfence.com/help/firewall/optimizing-the-firewall/?utm_source=plugin&utm_medium=pluginUI&utm_campaign=docsIcon#remove-optimization-manually
Scott

[image: ScreenShot.jpg]

On Thu, Apr 28, 2022 at 2:33 PM 97eighty ***@***.***> wrote:

> @sgibson71 <https://github.com/sgibson71> Great information. Thanks.
> WordFence has a throttling and rate limiting feature built in. It's
> possible this feature is causing problems. Can you review the settings to
> see if there are any limitations? Unfortunately, WordFence doesn't allow
> you to bypass for a specific plugin. But taking a look at these settings
> and adjusting may help or at least identify the cause. Take a look at this
> WordFence video on YouTube to understand and access this feature.
>
> https://youtu.be/9mtGKVZcZAU
>
> If this isn't the cause we'll have to do some more investigation.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/code4recovery/12-step-meeting-list/issues/791#issuecomment-1112533684>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASEV4O7N4OUJZ42UHUUCTA3VHLKVZANCNFSM5UPJAA4Q>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

*MOBILE:* (717) 333-4277
*WEB: *www.axiomwebworks.com
